istio/src/main/scala/io/buoyant/k8s/istio/IstioResponse.scala

```diff
@@ -0,0 +1,11 @@
+package io.buoyant.k8s.istio
+
+import com.twitter.util.Duration
+
+case class IstioResponse[Resp](statusCode: Int, duration: Duration, resp: Option[Resp]) extends IstioAttributesHolder {
+
+  def responseCode: ResponseCodeIstioAttribute = ResponseCodeIstioAttribute(statusCode)
+
+  def responseDuration: ResponseDurationIstioAttribute = ResponseDurationIstioAttribute(duration)
```

Comment:
once again, I'd really like to see some more line breaks in this file. I'd find something like
```scala
package io.buoyant.k8s.istio

import com.twitter.util.Duration

case class IstioResponse[Resp](
  statusCode: Int,
  duration: Duration,
  resp: Option[Resp]
) extends IstioAttributesHolder {

  def responseCode: ResponseCodeIstioAttribute = 
    ResponseCodeIstioAttribute(statusCode)

  def responseDuration: ResponseDurationIstioAttribute = 
    ResponseDurationIstioAttribute(duration)

}
```
much more readable. 

Note that this is entirely a matter of personal taste & I won't hold it against you :)

Reply:
This one is similar to the one above, but different in one important way. As I mentioned there, I don't really have a personal preference, all I would lobby for is for us to share a single formatter configuration—my general philosophy is that I usually don't care about these things as long as there's _one_ way to do it and we all follow, whatever this way might be.

All that said, my editor right now wraps at 120 columns and the Effective Scala guide mentions 100  columns as the standard: 
> Indent by two spaces. Try to avoid lines greater than 100 columns in length. Use one blank line between method, class, and object definitions.

What rule should we follow, wrap at 100, always wrap...?